docs/architecture/cloud-native/candidate-apps.md

```diff
@@ -68,7 +68,7 @@ With the introduction behind, we now dive into a much more detailed look at clou
 
 - [Cloud Native Computing Foundation](https://www.cncf.io/)
 
-- [.NET Microservices: Architecture for Containerized .NET applications](https://dotnet.microsoft.com/download/thank-you/microservices-architecture-ebook)
+- [.NET Microservices: Architecture for Containerized .NET applications](https://docs.microsoft.com/dotnet/standard/microservices-architecture/)
```

Comment:
earlier link is correct and helps in tracking downloads. Can you revert these changes?

Reply:
Done